### PR TITLE
Update ippeveprinter documentation

### LINKED ISSUE #744: `ippeveprinter` `-F output-type/subtype` argument doesn't do anything
**Is your feature request related to a problem? Please describe.**
The manual for ippeveprinter (https://openprinting.github.io/cups/doc/man-ippeveprinter.html) says:
```
-F output-type/subtype[,...]
    Specifies the output MIME media type. The default is "application/postscript" when the -P option is specified. 
```

and also...

```
the "OUTPUT_FORMAT" environment variable contains the output MIME media type
```

While I see the argument supplied to `-F` eventually make its way to the `OUTPUT_FORMAT` environment variable, this has no actual effect on the file produced. 

This StackOverflow post (https://stackoverflow.com/questions/69883878/setting-up-a-print-to-disk-system-with-ippeveprinter) suggests passing `-c ippeveps` to `ippeveprinter` to get format conversions working, but concedes that one is only able to output PostScript regardless of the arguments given to `ippeveprinter`.

**Describe the solution you'd like**
I would like the `-F` argument to control the format of the file output as indicated in the documentation.

**Describe alternatives you've considered**
I could use script something up using programs like ps2pdf and ImageMagick to convert the output into various image formats.

**Additional context**
The `OUTPUT_FORMAT` environment variable is filled by `ippeveprinter` in with the value of `-F` argument. I wrote a very short script:
```
env > /tmp/env
echo "$*" > /tmp/args
```
and passed it using `-c` to `ippeveprinter` to verify, eg., `ippeveprinter -F "some/thing" ...` yields `OUTPUT_TYPE=some/thing` in the `/tmp/env` file.

I searched the source of `ippeveps.c` (https://github.com/OpenPrinting/cups/blob/a8eaa67dd10dbaa7856b436ad921ad748f7de0bb/tools/ippeveps.c) and found no mentions of `OUTPUT_FORMAT`. Everything I see hard-codes output formats to PostScript.

### PR DESCRIPTION
**Pull request by zdohnal**
Fixes #744

### CODE REVIEW

File: doc/help/man-ippeveprinter.html [hunk 1]
```diff
@@ -110,11 +110,13 @@ <h2 class="title"><a name="OPTIONS">Options</a></h2>
 <b>ippeveprinter</b>
 will create an output file using the job ID and name.
 <dt><b>-F </b><i>output-type/subtype[,...]</i>
-<dd style="margin-left: 5.0em">Specifies the output MIME media type.
+<dd style="margin-left: 5.0em">Specifies the output MIME media type in case printer command (parameter "-c") supports more media types.
```
Comment (michaelrsweet) [inline doc/help/man-ippeveprinter.html:113]: How about "Specifies the output MIME media type that is provided to the printer command."?

File: doc/help/man-ippeveprinter.html [hunk 2]
```diff
@@ -110,11 +110,13 @@ <h2 class="title"><a name="OPTIONS">Options</a></h2>
...
 The default is "application/postscript" when the <b>-P</b> option is specified.
 <dt><b>-M </b><i>manufacturer</i>
 <dd style="margin-left: 5.0em">Set the manufacturer of the printer.
 The default is "Example".
+<dt><b>-K </b><i>keypath</i>
+<dd style="margin-left: 5.0em">Set location of server x.509 certificates and keys
```
Comment (michaelrsweet) [inline doc/help/man-ippeveprinter.html:119]: X.509 (capitalized)

File: man/ippeveprinter.1 [hunk 3]
```diff
@@ -118,13 +118,16 @@ When specifying a directory,
 will create an output file using the job ID and name.
 .TP 5
 \fB\-F \fIoutput-type/subtype[,...]\fR
-Specifies the output MIME media type.
+Specifies the output MIME media type in case printer command (parameter "\-c") supports more media types.
```
Comment (michaelrsweet) [inline man/ippeveprinter.1:121]: See above

File: man/ippeveprinter.1 [hunk 4]
```diff
@@ -118,13 +118,16 @@ When specifying a directory,
...
 The default is "application/postscript" when the \fB\-P\fR option is specified.
 .TP 5
 \fB\-M \fImanufacturer\fR
 Set the manufacturer of the printer.
 The default is "Example".
 .TP 5
+\fB\-K \fIkeypath\fR
```
Comment (michaelrsweet) [inline man/ippeveprinter.1:128]: See above

File: man/ippeveprinter.1 [hunk 5]
```diff
@@ -137,6 +140,9 @@ printer command ("\-c ippeveps").
 Specifies the maximum IPP version to report.
 2.0 is the default.
 .TP 5
+\fB-a \fIfilename.conf\fR
+Load printer attributes from conf file acquired by parameter "--ippserver" of command "ipptool". See "man ipptool" for more info.
```
Comment (michaelrsweet) [inline man/ippeveprinter.1:144]: Try:
This needs to be properly annotated for the cross-reference links to work. Also, the man page guidelines say to put one sentence per line. How about:

```
Loads printer attributes from "filename.conf".
Attribute files can be created using the
.BR ipptool (1)
command with the
.I \-\-ippserver
option.
```

**Review by michaelrsweet — CHANGES_REQUESTED** [review]
See comments.

File: man/ippeveprinter.1 (hunk 5 above)
Comment (zdohnal) [reply man/ippeveprinter.1]: Applied.

**Comment by zdohnal** [comment]
Thank you for the feedback! I've applied the changes.

**Review by michaelrsweet — APPROVED** [review]
LGTM

**Pull request merged**